src/scripts/sshvm

```diff
@@ -9,11 +10,11 @@ if '@' in n:
 else:
     user = ''
     host = n
-
+_
```

Comment:
I'm reading through the code. I think the underline above is ok, but it's not useful. Or do I miss something?